pb/countrycodes.pb.go

```diff
@@ -233,7 +233,7 @@ const (
 	CountryCode_SUDAN                          CountryCode = 209
 	CountryCode_SURINAME                       CountryCode = 210
 	CountryCode_SVALBARD                       CountryCode = 211
-	CountryCode_SWAZILAND                      CountryCode = 212
```

Comment:
I checked through the server codebase and we don't appear to reference by the `pb.CountryCode_SWAZILAND`, `pb.SWAZILAND` or indirectly through any accessor. Hopefully this won't bite us later.

Reply:
We need to recompile. Just changing the file will likely produce errors.